KsApi/models/UpdateBackingEnvelopeTests.swift

```diff
@@ -7,6 +7,7 @@ final class UpdateBackingEnvelopeTests: XCTestCase {
     {
       "updateBacking": {
         "checkout": {
+          "id": "2020",
```

Comment:
Since the `UpdateBackingEnvelope` now requires a `checkoutId` to be present for deserialization, you'll need to also add it to the `UpdateBackingMutation` as well so that deserialization doesn't break for update backing.